apps/browser/qml/pages/components/TabItem.qml

```diff
@@ -40,74 +49,102 @@ BackgroundItem {
     // contentItem is hidden so this cannot be children of the contentItem.
     // So, making them as siblings of the contentItem.
     data: [
-        Rectangle {
-            color: "#202020"
+        Item {
             width: root.implicitWidth
             height: root.implicitHeight
-        },
-        Image {
-            id: image
+            layer.effect: PressEffect {}
+            layer.enabled: _showPress
 
-            source: thumbnailPath
-            width: root.implicitWidth
-            height: root.implicitHeight
-
-            cache: false
-            asynchronous: true
-            opacity: status !== Image.Ready && source !== "" ? 0.0 : 1.0
-            Behavior on opacity { FadeAnimation {} }
-        },
-        Rectangle {
-            anchors.bottom: parent.bottom
-            width: parent.width
-            height: Math.max(close.height, titleLabel.height) + Theme.paddingLarge * 2
-
-            gradient: Gradient {
-                GradientStop { position: 0.0; color: "transparent" }
-                GradientStop { position: 0.45; color: Qt.rgba(0, 0, 0, Theme.opacityLow)}
-                GradientStop { position: 0.9; color: Qt.rgba(0, 0, 0, Theme.opacityOverlay)}
+            Rectangle {
+                color: "#202020"
```

Comment:
It would be better not to hard-code the colour here. Could this be made a function of one of the Theme colours?

For the colours in general, is it correct, according to the design, that the colours ignore the Theme/ambience? It would be nice to have the tab background, text and icon colours "inverted" for light ambiences.

Reply:
Reworked using ColorOverlay to get the color out of the atmosphere

Reply:
Thanks; this is nice.